package.json

```diff
@@ -122,7 +122,7 @@
     "chai-string": "^1.4.0",
     "cheerio": "^1.0.0-rc.3",
     "chokidar-cli": "latest",
-    "cypress": "3.1.5",
+    "cypress": "^3.8.0",
```

Comment:
I think we prefer to keep dependencies anchored to a specific version nowadays as dependabot will upgrade versions. So remove the `^` I think

Reply:
This happened automatically as a result of running `npm audit fix` but sure I'll remove it.

Reply:
looks like browser tests have failed, so we probably can't upgrade